### PR TITLE
Add Dockerfile for simple CKAN deployment

### LINKED ISSUE #1: Add Dockerfile for simple CKAN deployment.
This should allow deployment of a vanilla CKAN instance.

This can be based on CKAN's own [Dockerfile](https://github.com/ckan/ckan/blob/master/Dockerfile)

### PR DESCRIPTION
**Pull request by Mercy-Muchai**
Contribution

 Closes #1 

### Chore:
- [x] Update the README
- [x] Create docker-compose.yml file to allow deployment of a vanilla CKAN instance.

### CODE REVIEW

File: circle.yml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+general:
+  branches:
+    ignore:
+      - update_dockerfile # list of branches to ignore
```
Comment (DavidLemayian) [inline circle.yml]: I don't think we have this branch on here.. Also, Circle integration would be a separate PR for #11 ?

File: contrib/ckan/Dockerfile [hunk 2]
```diff
@@ -11,15 +10,18 @@ ENV CKAN_SITE_URL http://localhost:5000
...
         python-virtualenv \
         libpq-dev \
         git-core \
-	&& apt-get -q clean
+    && apt-get -q clean
 
-# Download CKAN
-RUN git clone https://github.com/ckan/ckan.git /ckan
+# Download CKAN		
+ RUN git clone https://github.com/ckan/ckan.git /ckan
```
Comment (DavidLemayian) [inline contrib/ckan/Dockerfile]: Any reason for the space/tabs before and on these lines?

File: contrib/ckan/Dockerfile [hunk 3]
```diff
@@ -36,8 +38,8 @@ RUN ckan-pip install --upgrade -r $CKAN_HOME/src/ckan/requirements.txt
 ADD ./dev-requirements.txt $CKAN_HOME/src/ckan/dev-requirements.txt
 RUN ckan-pip install --upgrade -r $CKAN_HOME/src/ckan/dev-requirements.txt
 
-# TMP-BUGFIX https://github.com/ckan/ckan/issues/3594
-RUN ckan-pip install --upgrade urllib3
+# TMP-BUGFIX https://github.com/ckan/ckan/issues/3594		
+ RUN ckan-pip install --upgrade urllib3
```
Comment (DavidLemayian) [inline contrib/ckan/Dockerfile]: Same as above.

File: circle.yml (hunk 1 above)
Comment (Mercy-Muchai) [reply circle.yml]: Will remove this file.

File: contrib/ckan/Dockerfile (hunk 2 above)
Comment (Mercy-Muchai) [reply contrib/ckan/Dockerfile]: Notes. Will edit that

File: contrib/ckan/Dockerfile [hunk 4]
```diff
@@ -11,15 +10,18 @@ ENV CKAN_SITE_URL http://localhost:5000
...
         python-pip \
         python-virtualenv \
         libpq-dev \
         git-core \
-	&& apt-get -q clean
+        && apt-get -q clean
 
 # Download CKAN
-RUN git clone https://github.com/ckan/ckan.git /ckan
+ RUN git clone https://github.com/ckan/ckan.git /ckan
```
Comment (DavidLemayian) [inline contrib/ckan/Dockerfile]: There's still a space before this line..

File: contrib/ckan/Dockerfile [hunk 5]
```diff
@@ -37,7 +39,7 @@ ADD ./dev-requirements.txt $CKAN_HOME/src/ckan/dev-requirements.txt
 RUN ckan-pip install --upgrade -r $CKAN_HOME/src/ckan/dev-requirements.txt
 
 # TMP-BUGFIX https://github.com/ckan/ckan/issues/3594
-RUN ckan-pip install --upgrade urllib3
+ RUN ckan-pip install --upgrade urllib3
```
Comment (DavidLemayian) [inline contrib/ckan/Dockerfile]: Same here..

File: contrib/ckan/docker-compose.yml [hunk 6]
```diff
@@ -0,0 +1,28 @@
+# docker-compose build && docker-compose up
+ckan:
+  container_name: ckan
+  image: ckan/ckan:latest
```
Comment (DavidLemayian) [inline contrib/ckan/docker-compose.yml]: This should be `codeforafrica/ckan:latest` - https://github.com/CodeForAfricaLabs/openAFRICA#updating-docker-image
